amoro-ams/src/main/java/org/apache/amoro/server/optimizing/ExecutingStageTask.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.amoro.server.optimizing;
+
+import org.apache.amoro.optimizing.RewriteFilesInput;
+import org.apache.amoro.optimizing.RewriteFilesOutput;
+import org.apache.amoro.server.dashboard.utils.OptimizingUtil;
+import org.apache.amoro.server.persistence.TaskFilesPersistence;
+import org.apache.amoro.shade.guava32.com.google.common.base.MoreObjects;
+
+import java.util.Map;
+
+public class ExecutingStageTask
+    extends StagedTaskDescriptor<RewriteFilesInput, RewriteFilesOutput, MetricsSummary> {
+
+  private String partition;
+
+  // only for mybatis and could be optimized by type handler afterward
+  public ExecutingStageTask() {}
+
+  public ExecutingStageTask(
+      long tableId, String partition, RewriteFilesInput input, Map<String, String> properties) {
+    super(tableId, input, properties);
+    this.partition = partition;
```

Comment:
Persistence is required, otherwise information will be lost after recover.

Reply:
Persistence logic has not been modified.
Please check it in TaskRuntime.persistTaskRuntime()